### PR TITLE
[5.1] upmerge 2023-12-17

### PR DESCRIPTION
**Pull request by Razzo1987**
Upmerge from 5.0 to 5.1

### CODE REVIEW

File: administrator/language/en-GB/install.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <extension client="administrator" type="language" method="upgrade">
 	<name>English (en-GB)</name>
 	<tag>en-GB</tag>
-	<version>5.0.1</version>
+	<version>5.0.2</version>
```
Comment (richard67) [inline administrator/language/en-GB/install.xml:5]: This change should not be merged up as the same file in the 5.1-dev branch has already the right version 5.1.0, see here: https://github.com/joomla/joomla-cms/blob/5.1-dev/administrator/language/en-GB/install.xml#L5

The same applies to other changes of this kind.

Comment (Razzo1987) [reply administrator/language/en-GB/install.xml:5]: Thanks, there are also the same PR from @LadySolveig (https://github.com/joomla/joomla-cms/pull/42535) so we use it and delete this one

Comment (richard67) [reply administrator/language/en-GB/install.xml:5]: You should check first if her PR is complete. Your changes 27 files, hers changes 15. If the difference of 12 completely comes from versions like this here, then hers is complete. I don't have the time now to check that myself.

Comment (LadySolveig) [reply administrator/language/en-GB/install.xml:5]: I made a new test-upmerge local and there are 11 files with merge conflicts that come from versions. I will check where the one different missing file comes from. Thank you @richard67

Comment (LadySolveig) [reply administrator/language/en-GB/install.xml:5]: Found it, the LanguageHelper was already changed in Alpha-1 and should not be included in this upmerge.

**Pull request closed without merging**